src/main/java/com/northconcepts/datapipeline/examples/parquet/WriteDatabaseViewToParquetFile.java

```diff
@@ -0,0 +1,44 @@
+package com.northconcepts.datapipeline.examples.parquet;
+
+import com.northconcepts.datapipeline.core.DataReader;
+import com.northconcepts.datapipeline.core.StreamWriter;
+import com.northconcepts.datapipeline.examples.database.DB;
+import com.northconcepts.datapipeline.jdbc.JdbcReader;
+import com.northconcepts.datapipeline.job.Job;
+import com.northconcepts.datapipeline.parquet.ParquetDataReader;
+import com.northconcepts.datapipeline.parquet.ParquetDataWriter;
+
+import java.io.File;
+import java.sql.Connection;
+
+public class WriteDatabaseViewToParquetFile {
+
+    private static final String PARQUET_FILE = "example/data/output/WriteAParquetFileUsingSchemaFromDatabaseView.parquet";
+
+    public static void main(String[] args) {
+        DB db = new DB(); // creates HSQL DB
+
+        Connection connection = db.getConnection();
+
+        db.executeFile(new File("example/data/input/user_information.sql"));
+        db.executeFile(new File("example/data/input/user_information_view.sql"));
+        
+        DataReader reader = new JdbcReader(connection, "SELECT * FROM users_with_lastname_as_doe")
+                .setAutoCloseConnection(false);
+
+        ParquetDataWriter writer = new ParquetDataWriter(new File(PARQUET_FILE));
+
+        writer.setSchema(connection, "SELECT * FROM users_with_lastname_as_doe");
```

Comment:
Can we set the schema using `ParquetDataWriter.setSchema(Connection, String, String, String)` instead of a query?

Reply:
Done